docs/scripts/util/checkLink.ts

```diff
@@ -14,6 +14,50 @@ export type LinkInfo = {
 };
 const requestedUrl: Set<string> = new Set();
 
+async function returnStatus({
+  href,
+  linkIdx,
+  pageIdx,
+  pageUrl,
+  statusCode,
+  tagName,
+  tagText,
+}: LinkInfo): Promise<LinkInfo> {
+  if ([...defaultGoodStatusCodes, 308].includes(statusCode)) {
+    /**
+     * If 301 and from 'https://docs.amplify.aws/', add a "/" and check again
+     * because 'https://docs.amplify.aws/' adds a "/" and return a 301 to all the links not ending with "/"
+     * e.g. https://docs.amplify.aws/lib/auth/getarted/q/platform/js should return 404
+     */
+    if (statusCode === 301 && href.startsWith('https://docs.amplify.aws/')) {
```

Comment:
can we store this url alongside the other constants rather than inlining it into the file?